pom.xml

```diff
@@ -14,9 +14,9 @@
   <packaging>hpi</packaging>
 
   <scm>
-    <connection>scm:git:ssh://github.com/jenkinsci/kubernetes-credentials-plugin.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/kubernetes-credentials-plugin.git</developerConnection>
-    <url>https://github.com/jenkinsci/kubernetes-credentials-plugin</url>
+    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+    <developerConnection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</developerConnection>
```

Comment:
Looks incorrect, `git://` is a readonly protocol. Shouldn't it be `ssh://git@` instead?

Reply:
I'm using what we are using in [other plugins](https://github.com/jenkinsci/github-branch-source-plugin/blob/master/pom.xml#L29).

Reply:
for connection, it's fine, but for developerConnection, it's not. The plugin you reference has `git@github.com` in developerConnection string which is implicitly ssh.

Reply:
```
    <scm>
        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
        <tag>HEAD</tag>
    </scm>
```